deploy/terraform/aws/README.md

```diff
@@ -7,7 +7,7 @@
 
 Set necessary variables by creating a file `ds-ami.pkrvars.hcl` and adding the following variables according to your own usage.
 
-```hcl
+```shel
```

Comment:
Why not keep hcl?

Reply:
keep hcl is ok to me, but we execute this command into the shell instead of terrafrom. so I change to shell.

Reply:
so, do you prefer to use `hcl` or `shell`, all look go for me

Reply:
Oh, both is okay on me, either.